src/initializers/database-setup.php

```diff
@@ -41,8 +42,12 @@ public function initialize() {
 		ORM::configure( 'mysql:host=' . \DB_HOST . ';dbname=' . \DB_NAME );
 		ORM::configure( 'username', \DB_USER );
 		ORM::configure( 'password', \DB_PASSWORD );
+		if ( \WPSEO_Utils::is_development_mode() ) {
```

Comment:
This block can be removed if the `Database_Logger`  is turned into an integration.

Reply:
Done